src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -698,12 +706,23 @@ default String chatWebhook() {
         return "";
     }
 
+    @ConfigItem(
+        keyName = "externalWebhook",
+        name = "External Webhook Override",
+        description = "If non-empty, external plugin messages are sent to this URL, instead of the primary URL",
```

Comment:
Are they by default sent here instead of primary URL, but the external plugin can still override the URL?

Reply:
That order is correct, I like your config description rewrite